### PR TITLE
.NET 7 updates for final release.

### PR DESCRIPTION
**Pull request by aslicerh**
- Set dockerfiles to use RH packages instead of MS packages.
- Adjust expected test versions.
- Change Microsoft.NET.Test.Sdk version to one that supports ppc.
- Update precompiled apps to work for all arches.

### CODE REVIEW

**Comment by tmds** [comment]
Thanks!

File: 7.0/build/test/asp-net-hello-world-envvar/test/test1/test1.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.5.0-preview-20221003-04" />
```
Comment (omajid) [inline 7.0/build/test/asp-net-hello-world-envvar/test/test1/test1.csproj:9]: It's also an option to use17.4.0, which also support ppc64le and isn't a preview version.

Comment (aslicerh) [reply 7.0/build/test/asp-net-hello-world-envvar/test/test1/test1.csproj:9]: It does? Good to know. I was planning to update everything when 17.5 comes out, but we can swap it to 17.4. It's just for our tests, so overall isn't a huge deal, imo.

Comment (tmds) [reply 7.0/build/test/asp-net-hello-world-envvar/test/test1/test1.csproj:9]: Maybe @dependabot will come round and suggest the update to 17.5 when it's out. See the open PRs we got: https://github.com/redhat-developer/s2i-dotnetcore/pulls. Not sure how this got enabled on our repo.

Comment (tmds) [reply 7.0/build/test/asp-net-hello-world-envvar/test/test1/test1.csproj:9]: I've merged them: https://github.com/redhat-developer/s2i-dotnetcore/pulls?q=is%3Apr+author%3Aapp%2Fdependabot.

**Pull request merged**